packages/veritone-react-common/src/components/InfiniteDirectoryList/styles.scss

```diff
@@ -0,0 +1,20 @@
+.loading-container {
```

Comment:
Use camel case class names. This allows easier access to styles (styles.loadingContainer).

`loading-container` => `loadingContainer`

Reply:
Actually, scss convention className is not using camelCase
http://bdavidxyz.com/blog/how-to-name-css-classes/